### PR TITLE
Fix untitled file opening

### LINKED ISSUE #9115: Untitled documents have different naming convention based on how created
- Azure Data Studio Version: `master`
 
Steps to Reproduce:

1. Create an untitled SQL editor using New Query command on node in OE
2. Create an untitled SQL editor using New Query command from File menu

Expected: Both documents follow same naming convention

Actual: One is named `SQLQuery_1` the other is named `Untitled-1`

Screen Shot 2020-02-11 at 10 28 46 AM

### PR DESCRIPTION
**Pull request by aaomidi**
This PR fixes #9115 

If this is accepted I'll cherrypick this for master as well.

### CODE REVIEW

**Comment by anthonydresser** [comment]
This is definitely not the right fix for master.

**Comment by aaomidi** [comment]
What would the right fix be?

File: src/vs/workbench/contrib/files/browser/fileActions.ts [hunk 1]
```diff
@@ -155,13 +155,14 @@ export class GlobalNewUntitledFileAction extends Action {
...
 		label: string,
-		@IEditorService private readonly editorService: IEditorService
+		@ICommandService private readonly commandService: ICommandService // {{ SQL CARBON EDIT }} - commandService to open newQuery
 	) {
 		super(id, label);
 	}
 
 	run(): Promise<any> {
-		return this.editorService.openEditor({ options: { pinned: true } }); // untitled are always pinned
+		// {{ SQL CARBON EDIT }} - open newQuery
```
Comment (Charles-Gagnon) [inline src/vs/workbench/contrib/files/browser/fileActions.ts]: Doesn't this break File -> New File though? It'd always open a sql query then.

Comment (aaomidi) [reply src/vs/workbench/contrib/files/browser/fileActions.ts]: No, that goes through another action:

https://github.com/microsoft/azuredatastudio/blob/97b9c43c7a7aaad46be061d41eb2fa89db1e86a7/src/vs/workbench/contrib/files/browser/fileActions.ts#L146

**Comment by anthonydresser** [comment]
We can do something in the query editor editor replacer class to replace the untitled input with one with the correct prefix. It will be more of an involved fix logic wise, so I don't have an issue with this for 1.15

**Comment by aaomidi** [comment]
I'm confused, what's wrong with this fix? It ensures all of our code goes through the same path.

**Comment by anthonydresser** [comment]
limiting edits to vscode source. This also won't catch other places that untitled's are created, where as make the change in the editor replacer would catch those.

**Comment by kburtram** [comment]
@aaomidi @anthonydresser we don't need this fix for Feb since I think this has been the behavior for several releases and we should try to avoid taking any more changes this month if possible.  Let's please iterate on the desired solution and take it to master for a future release.  Thanks!

**Pull request closed without merging**